control_plane/src/endpoint.rs

```diff
@@ -621,6 +621,11 @@ impl Endpoint {
         }
     }
 
+    pub fn sighup(&self) -> Result<()> {
+        self.pg_ctl(&["reload"], &None)?;
```

Comment:
look at VM monitor. It executes `pg_reload_conf`. Should we follow the way?

Reply:
I've changed it to pass through to `compute_ctl`.